python/tvm/relax/op/base.py

```diff
@@ -97,7 +97,11 @@ def call_tir(
     ret: Call
         A call node for the call_tir operator.
     """
-    if isinstance(args, Expr) and not isinstance(args, RxTuple):  # type: ignore
+    if (
+        isinstance(args, Expr)
+        and not isinstance(args, RxTuple)
+        and not isinstance(args.struct_info_, TupleStructInfo)
```

Comment:
You should do this for the other call_tir variants in this file too.

Reply:
Good call, and updated.